test/unit/lib/impl/AzureBlobTvm.test.js

```diff
@@ -12,6 +12,10 @@ governing permissions and limitations under the License.
 
```

Comment:
Two things:
- Could you add test to make sure it calls `azureUtil.addAccessPolicyIfNotExists` with correct params here.
- in AzurePresignTvm.test, make sure `azureUtil.getPolicy` is called and mock it to return a `policyId` + check the return value to make sure the correct `policyId` is returned